src/openapi.json

```diff
@@ -95,6 +95,12 @@
 						"example": "upload.csv",
 						"pattern": "^.+\\.csv$"
 					},
+					"fileSize": {
```

Comment:
I don't think we document fields in our swagger spec but this one might be worth adding a clarification that the file size is in bytes.

Reply:
I do think that would be a good idea! Looks like we can add the `description` attribute, although its discoverability is pretty poor.

https://github.com/PhilanthropyDataCommons/service/assets/4731/a7802531-9789-4363-8903-536c5815f741